addons/website_sale/tests/test_website_sale_show_compare_list_price.py

```diff
@@ -75,8 +75,8 @@ def setUpClass(cls):
                 Command.create({
                     'applied_on': '1_product',
                     'product_tmpl_id': cls.test_product_with_pricelist_and_compare_list_price.id,
-                    'compute_price': 'fixed',
-                    'fixed_price': 3500,
+                    'compute_price': 'percentage',
```

Comment:
why do we need to change test cases? :thinking:

Reply:
These test cases have fixed prices which is checked in the tour to trigger and click, but due to our specification, we now only show strikethrough when compute_price is percentage. Hence for the tour we had 2 approaches:
1. Either remove the part where we checked the base_price as it will not be shown when we use fixed price or formula
2. Adapt the tour accordingly where we show percentage price which will be visible for strikethrough.

Reply:
IMO we should have test cases for both flows
- for existing ``fixed` compute price adept tour 
- Add another tour for `percentage`  with strike-through

To ensure we are not breaking fix compute price flow

Reply:
Don't spend too much time on adding new tests here, adapting them is good already for now, unless you really notice an issue that wasn't caught by the test when it should have.
Since this is a functional change (even if it makes sense), it should be deployed quickly to avoid users becoming used to the current behavior.